AODN/AODN-WAVE-NRT/ARDC_SOFAR_API_NRT/config/variables_lookup.csv

```diff
@@ -0,0 +1,12 @@
+id,sofar_variable_name,aodn_variable_name
+0,meanPeriod,WPFM
+1,peakPeriod,WPPE
+2,meanDirection,SSWMD
+3,peakDirection,WPDI
+4,meanDirectionalSpread,WMDS
+5,peakDirectionalSpread,WPDS
+6,significantWaveHeight,WSSH
+7,latitude,LATITUDE
+8,longitude,LONGITUDE
+9,timestamp,TIME
+
```

Comment:
Hey @lbesnard, so these are all the variables in the real-time file correct? I am editing the netcdf template and will make sure I keep only the variables for real-time

Reply:
They are the variables I have found in the data downloaded from the API for the only token I was given

Reply:
Which token is this and who gave to you?